app/src/main/res/layout-v21/login.xml

```diff
@@ -140,6 +140,18 @@
             android:visibility="gone" />
 
         <TextView
+            android:id="@+id/register"
```

Comment:
Its generally a best practice to group common attributes values in xml tags in styles https://developer.android.com/guide/topics/ui/themes.html. Since Register, Another Server and Legal Info textviews are pretty similar we can group their common xml attributes in a style.